config/v1/types_build.go

```diff
@@ -42,6 +42,11 @@ type BuildSpec struct {
 	// BuildOverrides controls override settings for builds
 	// +optional
 	BuildOverrides BuildOverrides `json:"buildOverrides"`
+
+	// BuildCSIVolumes defines whether the instances of DockerBuildStrategy or SourceBuildStrategy
+	// can employ CSI volumes with any BuildVolume they specify.  Currently, this is only
+	// supported on clusters where the TechPreview feature set is enabled.
+	BuildCSIVolumes bool `json:"buildCSIVolumes,omitempty"`
```

Comment:
@deads2k do we typically prefix things like this with `Enable` or do we explicitly avoid that?

@gabemontero not sure what happened w/ the formatting here, but can you get the whitespace lined up in this struct?  I'm surprised this isn't failing gofmt validation.

Reply:
just ran `gofmt -w ./config ./pkg` and `gofmt -w -l ./config ./pkg`it produced no changes @bparees 

also examined tab vs. spaces in the preceding lines with the new ones and they are consistent

not sure why git's diff is complaining

Reply:
also, I see no difference when I look at the code diffs from the PR in my browser @bparees

Reply:
bizarre, this is what i see in the browser:
![image](https://user-images.githubusercontent.com/4974046/132707118-af216bcb-8eaf-4715-8f72-ec48395264a6.png)

but yeah, doesn't seem to be reflected in the file itself.

Reply:
on the naming convention, fwiw, I did look at the https://docs.openshift.com/container-platform/4.8/rest_api/understanding-compatibility-guidelines.html link you provided on the email list recently, and did not see anything that fit with what you are asking about